Makefile

```diff
@@ -21,3 +21,9 @@ tests:
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos5 ./run_unit_tests.sh
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos6 ./run_unit_tests.sh
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos7 ./run_unit_tests.sh
+
+lint:
+	bash -c "[[ ! -z ./convert2rhel/ ]] && find ./convert2rhel/ -name '*.py' | sort -u | xargs pylint --rcfile=.pylintrc"
```

Comment:
What benefits does this provide as opposed to `pylint convert2rhel/`?

Reply:
You need to pass `pylint --rcfile=.pylintrc convert2rhel`  or just `make lint`

The goal here is to have `make lint` to show all lint messages and `make lint-errors` top show only the errors

Reply:
Right, but why do we need to use so many commands and pipe into pylint? Why can't this line simply be `pylint convert2rhel/`?